src/ContractEditor/plugins/withClauses.js

```diff
@@ -129,7 +128,7 @@ const withClauses = (editor, withClausesProps) => {
         return editor;
       } catch (err) { console.error(err); }
     }
-    insertData(data);
+    return insertData(data);
```

Comment:
Was copy/paste not working?

Reply:
It was. This was just to fix a linting warning: https://eslint.org/docs/rules/consistent-return

Reply:
I would think this would break it then, because we don't call `return editor` at the end of `editor.insertData`

Reply:
it doesn't break it. Before this change, the function returns undefined if we get to this line since we don't explicitly call return in this case. We could explicity call return on the next line  which will also return undefined. I also don't see why we need to return `editor` on line 128, so we could just return nothing so the function always returns undefined in both cases and is therefore consistent. Any of these are solutions to fix the linter, but none will change the behavior since it doesn't matter what we return from this function.

Reply:
I'm going to go with removing line 128. `editor` should only be returned on line 145